package.json

```diff
@@ -10,8 +10,12 @@
   },
   "dependencies": {
     "@deriv/deriv-api": "^1.0.9",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@fortawesome/react-fontawesome": "^0.1.18",
     "@xstate/react": "^3.0.0",
     "dialog-polyfill": "^0.5.6",
+    "font-awesome": "^4.7.0",
```

Comment:
dont use libraries for the arrows, we have arrows in the img folder as svg, pls use them